Update README.md with Coding Standards link (#9793)
RMC-14/RMC-14: README.md

## Contributing Requirements
- Understanding how to contribute - Read this resource provided by SS14 and attempt to keep to it. [Pull Request and Changelog Guidelines](https://docs.spacestation14.com/en/general-development/codebase-info/pull-request-guidelines.html).

> - Standards - Following the commonly agreed upon standards, ensuring that works are submitted at a reasonable state. [Coding Standards](coding-standards.md)

Reviewer: Tiny suggestion:
```suggestion
- Standards - Following the commonly agreed upon standards, ensuring that work is submitted in a reasonable state. [Coding Standards](coding-standards.md)
```
